static/.htaccess

```diff
@@ -14,7 +14,10 @@ RewriteRule ^docs(/(8.0|next))?/components/modeler/desktop-modeler/element-templ
 # Added Self-Managed Modeler redirect
 RewriteRule ^docs/next/self-managed/web-modeler/installation/(.*)$ /docs/next/self-managed/modeler/web-modeler/installation/$1 [R=301,L]
 
-RewriteRule ^docs/next/apis-clients/working-with-apis-clients/?$ /docs/next/apis-clients/working-with-apis-tools/ [R=301,L]
+# Added redirect rules for APIs/Clients to APIs/Tools
+RewriteRule ^docs/next/apis-clients/working-with-apis-clients/?$ /docs/next/apis-tools/working-with-apis-tools/ [R=301,L]
+RewriteRule ^docs/next/apis-clients(.*)$ /docs/next/apis-tools$1 [R=301,L]
+RewriteRule ^optimize/next/apis-clients(.*)$ /optimize/next/apis-tools$1 [R=301,L]
```

Comment:
notes, non-blocking:

1. The first rule needs to be kept because of the change in file name (@christinaausley I missed that when we were rewriting them, and added it back!)
2. The second two rules are wildcards that will send any page within the `apis-clients` "folder" to the `apis-tools` "folder.
3. We are only applying these to vNext right now, and will add more as the renaming propagates to the other versions.